core/src/components/toggle/toggle.scss

```diff
@@ -240,8 +240,6 @@ input {
 
   background: var(--track-background);
 
-  pointer-events: none;
```

Comment:
This is removed from the `.toggle-icon` element so users can swipe on the element.